draft-ietf-webpush-protocol.xml

```diff
@@ -430,18 +430,24 @@ Location: https://push.example.net/subscription/i-nQ3A9Zm4kgSWg8_ZijV
         An application server requests the delivery of a push message by sending
         a HTTP POST request to a push resource distributed to the application
         server by a user agent. The ciphertext of the push message is included
-        in the body of the request.
+        in the body of the request. (Note that the example below shows a body
+        that is base64 encoded and lines are wrapped. This is due to formatting
+        constraints; an encrypted push message message will probably contain
+        binary data.)
       </t>
       <figure>
         <artwork type="inline">
   <![CDATA[
 POST /push/JzLQ3raZJfFBR0aqvOMsLrt54w4rJUsV HTTP/1.1
 Host: push.example.net
 TTL: 15
-Content-Type: text/plain;charset=utf8
-Content-Length: 36
+Content-Length: 33
+Encryption: keyid="dhkey"; salt="Qg61ZJRva_XBE9IEUelU3A"
+Crypto-Key: keyid="dhkey";
+                dh="BDgpRKok2GZZDmS4r63vbJSUtcQx4Fq1V58-6-3NbZzS
+                    TlZsQiCEDTQy3CZ0ZMsqeqsEb7qW2blQHA4S48fynTk"
 
-iChYuI3jMzt3ir20P8r_jgRR-dSuN182x7iB
+yqD2bapcx14XxUbtwjiGx69eHE3Yd6AqXcwBpT2Kd1uy

```

Comment:
[BINARY DATA] ? 

It's confusing because in a previous (temporary) implementation we actually required
the content to be base64 encoded - the text above should clear this out, but I don't see
any benefit in having the b64 text.